Add exception info for `HTMLCanvasElement.transferControlToOffscreen()` and `OffscreenCanvas.transferToImageBitmap()` (#36631)
mdn/content: files/en-us/web/api/offscreencanvas/transfertoimagebitmap/index.md

If you call `transferToImageBitmap()` and don't intend to pass it to {{domxref("ImageBitmapRenderingContext.transferFromImageBitmap()")}}, consider whether you need to call `transferToImageBitmap()` at all. Many web APIs which accept `ImageBitmap` also accept `OffscreenCanvas` as an argument.

### Exceptions

- `InvalidStateError` {{domxref("DOMException")}}

>   - : Throws if the canvas has transferred to another context scope, for example, to worker; or hasn't set to a context mode via calling {{domxref("OffscreenCanvas.getContext()")}}.

Reviewer: ```suggestion
  - : Throws if:
      - the canvas has transferred to another context scope, such as a worker
      - the canvas context mode has not been set by calling {{domxref("OffscreenCanvas.getContext()")}}.
```
